docs/getting-started/quick-start.md

```diff
@@ -3,28 +3,97 @@ category: installation
 order: 10
 menu-title: Quick Start
 meta-title: Quick Start | CKEditor 5 documentation
-meta-description: Learn how to install, integrate, configure, and develop CKEditor 5. Browse through the API documentation and online samples.
+meta-description: Learn the fastest way to install a powerful, rich text WYSIWYG editor - CKEditor 5 - in your web application using npm.
 ---
 
-# Getting started with CKEditor&nbsp;5
+# Quick Start
 
-CKEditor&nbsp;5 provides every type of WYSIWYG editing solution imaginable. From editors similar to Google Docs and Medium to Slack or Twitter-like applications, all is possible within a single editing framework. It is an ultra-modern JavaScript rich-text editor with MVC architecture, custom data model, and virtual DOM, written from scratch in TypeScript with excellent webpack and Vite support. Find out the most convenient way to start using it!
+CKEditor&nbsp;5 is a powerful, rich text editor you can embed in your web application. This guide will show you the fastest way to start using it.
 
-<span class="navigation-hint_mobile">
-	<info-box>
-		Use the **main menu button in the upper-left corner** to navigate through the documentation.
-	</info-box>
-</span>
+## Try CKEditor&nbsp;5 builder
 
-<!-- the wrapper class should be transformed into an additional infobox class if we will want to use it on other pages-->
-<!-- <div class="info-cta"> -->
-<info-box>
-	**Migrating from CKEditor 4?** {@link updating/migration-from-ckeditor-4 Check the migration section}
-</info-box>
+Check out our interactive builder to quickly get a taste of CKEditor&nbsp;5. It offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs.
+
+## Installing CKEditor&nbsp;5 using npm
 
 <info-box>
-	**Want to remove the "Powered by CKEditor" logo?** {@link support/managing-ckeditor-logo Check the dedicated guide}
+
+**Prerequisites:**
+
+* [Node.js](https://nodejs.org/) in version 18 or above.
+* An initialized Node.js project.
```

Comment:
I'm nitpicking, but I wonder if someone from outside JS world, will understand this. What do we mean by Node.js project? Maybe we could be a bit more descriptive, like a folder where JavaScript dependencies will be installed, or just show the snippet as a bit longer (as [Remix](https://remix.run/docs/en/main/start/quickstart#installation)).

```shell
mkdir ckeditor5-test
cd ckeditor5-test
npm init -y
npm install ckeditor5
```

Reply:
Considering that users rarely read carefully, the snippet may be better.

Reply:
I think it's reasonable to require an existing Node.js project if someone is using **this** installation method. How many people will actually create a whole new project to use CKEditor?

I'd skip the prerequisites section entirely and go straight to `npm install ckeditor5`. This is what most libraries do, and IMO it will be less confusing and more straightforward for the majority of developers.

Examples:
https://handsontable.com/docs/javascript-data-grid/installation/
https://supabase.com/docs/reference/javascript/installing
https://www.chartjs.org/docs/latest/getting-started/installation.html
https://rollupjs.org/introduction/
https://orm.drizzle.team/docs/get-started-sqlite#better-sqlite3

Reply:
Sounds good!

Reply:
I forgot that we don't have a framework like Remix or Next.js. CKEditor is a framework, sure. But it's more like a package added to an existing project. I agree that few people will start from scratch, contrary to Remix or Next.js.